marimo/_snippets/data/duckdb-6.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2024 Marimo. All rights reserved.
+
+import marimo
+
+__generated_with = "0.11.0"
+app = marimo.App()
+
+
+@app.cell
+def _(mo):
+    mo.md(
+        r"""
+        # DuckDB: Data Export & Integration with Visualization
```

Comment:
This example feels out of place, it's really just an altair example. Let's remove it?